### DISCUSSION TITLE
Image upload issue

### DISCUSSION THREAD

**Discussion by productstrioangle** [rebing/graphql-laravel#884] [General]
I have get repeated errors on my laravel-graphql image upload functionality,

Please anyone can share the exact method of image upload

{
  query: `
    mutation($file: Upload!) {
      singleUpload(file: $file) {
        id
      }
    }
  `,
  variables: {
    file: File // a.txt
  }
}

**Comment by mfn** [comment]
Do you still have the problem? Did you follow https://github.com/rebing/graphql-laravel#file-uploads?

It would be great if you could show more actual code and the full error.

Also, there a couple of upload tests in the `test/`, might be worth checking out.

**Reply by lucasantoniooficial** [reply]
The question that is not so clear in the documentation

is $args['file'] a Laravel Request File or do I need to use the Storage class?

**Reply by mfn** [reply]
AFAIK it's `\Illuminate\Http\File`, you can use the `dump()` helper to be sure.
